package-lock.json

```diff
@@ -1,8 +1,14991 @@
 {
   "name": "root",
   "version": "0.14.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
You have the wrong version of NPM installed. lockfileVersion should be `1`. Please use Node v14.19.0 and NPM v6